static/templates/admin/editFilenameForm.html

```diff
@@ -0,0 +1,18 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="edit_filename_form" method="post" action="editfilename/<%= pid %>" enctype="multipart/form-data" target="upload_file_container" class="edit_form edit_filename_form">
```

Comment:
You will need to change the action here back to "editlabel/<%= pid %>" since that is the mapping being used by the controller server side
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/master/admin/src/main/java/edu/unc/lib/dl/admin/controller/EditLabelController.java#L27